libnet/configure.ac

```diff
@@ -309,11 +331,15 @@ AC_ARG_ENABLE([samples],
 
 AM_CONDITIONAL([INSTALL_SAMPLES], [test x"$enable_samples" = xyes])
 
+AC_ARG_ENABLE([pkgconfig],
+    [AS_HELP_STRING([--enable-pkgconfig],[generate pkg-config .pc file @<:@default=no@:>@])])
+AM_CONDITIONAL([ENABLE_PKGCONFIG], [test x"$enable_pkgconfig" = xyes])
+PKG_INSTALLDIR
```

Comment:
I took your fork for a spin (aka cloned and build it) earlier.
It sadly didn't last very long. I encountered one "not so huge" error during bootstrapping and a fatal error during configure.
During `autoreconf --install` I got this "error":

```
Makefile.am:43: error: 'pkgconfig_DATA' is used but 'pkgconfigdir' is undefined
autoreconf: automake failed with exit status: 1
```

Automake encountered the error, but the problem is here in configure.ac
This guy `PKG_INSTALLDIR`, seems lost.
I replaced this line with this and the error went away: (It's not mine. I saw other projects doing it like this)

`m4_ifdef([PKG_INSTALLDIR], [PKG_INSTALLDIR], AC_SUBST([pkgconfigdir], ${libdir}/pkgconfig))
`
Can you check if this is correct?